contents/docs/product-analytics/correlation.mdx

```diff
@@ -10,52 +10,40 @@ availability:
   enterprise: full
 ---
 
-Correlation analysis is a feature within [funnels](/docs/product-analytics/funnels) that **automatically** highlights significant factors impacting the conversion rate of users within a funnel.
+Correlation analysis is a feature within [funnels](/docs/product-analytics/funnels) that **automatically** highlights significant factors impacting conversion rate.
 
-Funnels have the ability to directly compare conversion rates of users for a specific property via the breakdown feature, but this requires you to know what property to investigate. If you're not certain, correlation analysis is helpful because it can highlight significant events or properties automatically. 
+Funnels have the ability to directly compare conversion rates of users for a specific property via the breakdown feature, but this requires you to know which property to investigate. If you're not certain, correlation analysis highlights significant events or properties for you. 
 
-Given a funnel, PostHog will calculate the conversion rate for different slices (either by events performed, properties they have or properties for the events they performed) and surface relevant signals. Below is an example of what correlation analysis might look like:
+As an example, the below insight shows that desktop users were `1.3x more likely` to convert than mobile or tablet users. Similarly, we also seee that users who completed the `uploaded_cv_for_review` event were `8.0x more likely` to convert.
```

Comment:
I think this can be slightly confusing, see: https://posthog.slack.com/archives/C0368RPHLQH/p1711479367617279
```suggestion
As an example, the below insight shows that desktop users were `1.3x more likely` to convert than mobile or tablet users. We also see that users who completed the `uploaded_cv_for_review` event while active in the funnel were `8.0x more likely` to convert.
```